boards/arc/emsdp/arc_mpu_regions.c

```diff
@@ -43,7 +43,7 @@ static struct arc_mpu_region mpu_regions[] = {
 	/* Region Peripheral */
 	MPU_REGION_ENTRY("PERIPHERAL",
 			 0xF0000000,
-			 64 * 1024,
+			 17 * 1024 * 1024,
```

Comment:
Sorry for nitpicking, but in our inherently power-or-f-2 digital world, 17 looks veeery unusual at least ;)